### PR TITLE
Drop support for EOL Python

### LINKED ISSUE #13: Python 2.6 is deprecated
As seen in the build at https://travis-ci.org/mswart/pyopenmensa/jobs/350746974#L443, there is a deprecation warning for Python 2.6.

I would suggest removing support and build for that version.

### PR DESCRIPTION
**Pull request by hugovk**
Fixes #13.

### CODE REVIEW

File: tests/feed/test_xml.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def tag(name):
 
 def find_child(parsed, tag_name, is_openmensa_tag=True):
     tag_name = tag_name if not is_openmensa_tag else tag(tag_name)
-    element = parsed.find("./{0}".format(tag_name))
+    element = parsed.find("./{}".format(tag_name))
     if element is None:
         raise KeyError("Element named {0!r} not found.".format(tag_name))
```
Comment (j-maas) [inline tests/feed/test_xml.py]: For consistency I would suggest changing this to `"Element named {!r} not found."...` as well.

**Review by j-maas — CHANGES_REQUESTED** [review]
Very comprehensive PR. :+1:

File: tests/feed/test_xml.py (hunk 1 above)
Comment (hugovk) [reply tests/feed/test_xml.py]: Updated!

**Comment by j-maas** [comment]
@mswart What is your opinion on this? :)

**Comment by hugovk** [comment]
Closing due to lack of interest. Please let me know if you'd like me to re-open it or update it.

**Pull request closed without merging**